### ISSUE TITLE
problem installing it

### ISSUE THREAD

**Issue by zewpo** [denolib/typeorm#119]
**Issue type:**

[*] bug report

**TypeORM version:**

[*] `latest` master

**Deno version:**

`1.4.1`

 >  deno --version
deno 1.4.1
v8 8.7.75
typescript 4.0.2

**Steps to reproduce or a small repository showing the problem:**

```
deno install --allow-read --allow-write --allow-net --allow-env -f -n typeorm https://deno.land/x/typeorm/cli.ts
.....
....
Check https://deno.land/x/typeorm@v0.2.23-rc8/cli.ts
error: TS2611 [ERROR]: 'queryRunner' is defined as a property in class 'EntityManager', but is overridden here in 'MongoEntityManager' as an accessor.
    get queryRunner(): MongoQueryRunner {
        ~~~~~~~~~~~
    at https://deno.land/x/typeorm@v0.2.23-rc8/src/entity-manager/MongoEntityManager.ts:80:9
```

**Comment by uki00a** [comment]
This problem has been fixed at [v0.2.23-rc9](https://github.com/denolib/typeorm/releases/tag/v0.2.23-rc9). Thank you for the report!

**Comment by zewpo** [comment]
thanks;
confirm success
...
Check https://deno.land/x/typeorm@v0.2.23-rc9/cli.ts
✅ Successfully installed typeorm
